contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java

```diff
@@ -50,17 +48,21 @@
  * @since 2.1.0
  */
 @Evolving
-public class CassandraPOJOOutputOperator extends AbstractCassandraTransactionableOutputOperatorPS<Object> implements Operator.ActivationListener<Context.OperatorContext>
+public class CassandraPOJOOutputOperator extends AbstractCassandraTransactionableOutputOperatorPS<Object>
```

Comment:
Can you check why we have almost similar abstract classes: AbstractCassandraTransactionableOutputOperator and AbstractCassandraTransactionableOutputOperatorPS ?